recipes/vembrane/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: vembrane
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/v/vembrane/vembrane-{{ version }}.tar.gz
+  sha256: 6653729d6a71eead9c9c5213661e749fbcd23d0b460e07f8b466d3db2e1023ec
+
+build:
+  number: 0
+  entry_points:
+    - vembrane = vembrane:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
```

@tedil, FYI:
If you wanted to use the GitHub release/tag directly, all you should have to do is:
```suggestion
    - pip
    - poetry >=0.12
```
;)